package.json

```
@@ -1,6 +1,6 @@
 {
     "name": "hrb-temaki",
-    "version": "0.11.14",
+    "version": "0.11.15",
```

MUST: 
今0.11.16なので、

```suggestion
    "version": "0.11.17",
```